src/Engine/UnitActionLifecycles/CaptureLifecycle.java

```diff
@@ -111,11 +111,12 @@ public GameEventQueue getEvents(MapMaster map)
           if( capture.willCapture() ) // If this will succeed, check if the CO will lose as a result.
           {
             // Check if capturing this property will cause someone's defeat.
-            if( (propertyType == TerrainType.HEADQUARTERS) && (null != captureLocation.getOwner()) )
+            if( Utils.willLoseFromLossOf(map, captureLocation) )
             {
               // Someone is losing their big, comfy chair.
               CommanderDefeatEvent defeat = new CommanderDefeatEvent(captureLocation.getOwner());
-              defeat.setPropertyBeneficiary(actor.CO);
+              if( (propertyType == TerrainType.HEADQUARTERS) )
+                defeat.setPropertyBeneficiary(actor.CO);
```

Comment:
So capturing labs won't award properties, only HQs?

Reply:
That is AWBW lab semantics, so yes.
Don't ask me why they do it that way.